### DISCUSSION TITLE
1.0.0

### DISCUSSION THREAD

**Discussion by mrkai77** [mrkai77/Loop#407] [Announcements]
## 📰 Important News
- 💸 Reminder to support me & Loop by [sponsoring the project](https://github.com/sponsors/MrKai77)!
- 💄 Can't donate? The next best way to support the project is to make some **app icons** for Loop!
- 🌐 **Localization:** I will start requesting localizations for the next update so please let me know if you are interested in localizing Loop :)

## ✨ New Features
- ✨ #361 Completely new translucent settings window (@Soft-Bred)
- ✨ #303 Set up localization infrastructure (@KrLite)
- ✨ #344 New website for release notes (@SenpaiHunters)
- ✨ #251 Ability to ignore mouse screen (@MrKai77)
- ✨ #235 Save cycle keybind index (@MrKai77)
- ✨ #317 Make window snapping less sensitive (@MrKai77)
- ✨ #212 Don't activate Mission Control when window snapping (@MrKai77)
- ✨ #298 Ability to animate window resizes when preview is disabled (@MrKai77)
- ✨ #298 Window animations no longer require screen recording access (@MrKai77)
- ✨ #378 Add summer/developer app icons (@N-coder82)
- ✨ #379 Ability to ignore screen with mouse (@MrKai77)
- ✨ #380 Reduce useless cycle/custom action creation click (@MrKai77)
- ✨ #384 Ability to move cursor with window (@MrKai77)
- ✨ #389 Improve window animations when window snapping (@MrKai77)
- ✨ #390 Ability to translate window location (@MrKai77)

## 🐞 Bug Fixes
- 🐞 #299 Instant animation is not instant (@MrKai77)
- 🐞 #347 Fix window being cropped to screen when restoring initial frames (@MrKai77)
- 🐞 #95 Windows will now properly resize to bottom with window animations (@MrKai77)
- 🐞 #350 Make window snapping work properly on secondary screens (@MrKai77)
- 🐞 #314 Reduce Hyperkey conflicts (@MrKai77)
- 🐞 #381 Make sure settings window buttons are themed properly (@MrKai77)

## 🥳 Notable Mentions
- 🐞 #296 Fix typo (@mrs1669)
- 🎨 #304 Reduce SwiftLint warnings (@phlpsong)
- 🎨 #370 Use SwiftFormat rather than SwiftLint (@MrKai77)
- ✨ #387 Add Luminare as git submodule (@MrKai77)

**Full Changelog**: https://github.com/MrKai77/Loop/compare/1.0.0-beta.15...1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/MrKai77/Loop/releases/tag/1.0.0'>1.0.0</a>.</em>

**Comment by JaceThings** [comment]
YIPPIE!!!!

**Comment by wjfu99** [comment]
The new UI design really surprise me, but the middle click trigger seems not work well, degrade to previous version can fix it.

**Comment by raphaelsalaja** [comment]
Nice work!

**Comment by Owen5123** [comment]
能否添加中文

**Comment by KrLite** [comment]
Hooray!

However, **Luminare,** the new UI library of Loop, seems to be private and missing localizations. It looks pretty but seems a little bit black boxed. I would be happy to know more of it and (if possible) contribute to localize Loop again.

**Reply by mrkai77** [reply]
Luminare doesn't contain any strings, and all the text is managed by Loop itself. I ended up having to remove the old Simplified Chinese localization for this release as the new settings window introduced a lot of new text to be localized, and the whole app became half-english, half simplified Chinese, which didn't make sense :)

Localization for Simplified Chinese is currently tracked in #319.

**Reply by mrkai77** [reply]
Luminare is now open-source at https://github.com/MrKai77/Luminare :D

**Comment by w1png** [comment]
I would love to help with Russian translation!

You can contact me via twitter.
